app/code/Magento/Sitemap/Model/Sitemap.php

```diff
@@ -159,7 +159,7 @@ class Sitemap extends \Magento\Framework\Model\AbstractModel implements \Magento
      * @var string
      * @since 100.2.0
      */
-    protected $_cacheTag = true;
+    protected $_cacheTag = [Value::CACHE_TAG];
```

Comment:
Looks like `@var` shoud NOT be a string.

Reply:
Thanks, you are right. According to the [coding standards](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html#inheritdoc) the doc should probably look like:

```PHP
/**
 * @inheritdoc
 *
 * @since 100.2.0
 */
protected $_cacheTag = [Value::CACHE_TAG];
```

By the way ... is there a guideline when the `@since` annotation should be used? Should it be preserved or removed?

Reply:
Updated phpdoc